Tests/Services/AWSLaunchWizardTests/EndpointResolverTest.swift

```diff
@@ -6,17 +6,10 @@ import AWSClientRuntime
 @testable import AWSLaunchWizard
 ========
```

Comment:
Not sure what's going on with this file, it has a lot of merge conflict markers in it.  Again, this is a generated file that you can leave out of this PR.

Just check it out from `main` and commit:
```
$ git checkout main -- Tests/Services/AWSLaunchWizardTests/EndpointResolverTest.swift
```

Reply:
It's now left out of the PR.